src/iGenVar.cpp

```diff
@@ -49,6 +49,9 @@ void initialize_argument_parser(seqan3::argument_parser & parser, cmd_arguments
     parser.add_option(args.threads, 't', "threads",
                       "Specify the number of decompression threads used for reading BAM files.",
                       seqan3::option_spec::standard);
+    parser.add_flag(args.verbose, 'v', "verbose",
+                    "If you set this flag to true, we provide additional details about what iGenVar does. The detailed "
+                    "output is printed in the standard output.");
```

Comment:
Not true, you use debug_stream, which prints to stderr.
And stderr is good, because otherwise the output would be intertwined with the vcf output (which goes to stdout).

Reply:
Instead of `If you set this flag to true,` just `If you set this flag`, because the call is `./program --verbose` without `true` as a parameter.